src/unistd.rs

```diff
@@ -1337,15 +1337,18 @@ pub fn setgid(gid: Gid) -> Result<()> {
 pub fn getgroups() -> Result<Vec<Gid>> {
     // First get the number of groups so we can size our Vec
     let ngroups = unsafe { libc::getgroups(0, ptr::null_mut()) };
```

Comment:
What I meant is that it would be more idiomatic to write it like this:
```rust
let ngroups = unsafe { libc::getgroups(0, ptr::null_mut()) }? as usize;
```